### DISCUSSION TITLE
Collage setup

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#201] [Q&A] [answered]
it is me again :-)

I play around with collage. I found how to remove the default image in the middle .
I add an aditional Colum in the config for a 4th picture. How can I remove one single config for an image in the collage

**Comment by mgineer85** [comment, marked as answer]
This is fixed in v2 (not released yet)
For v1.x I fear you need to edit the config json directly, remove the line and restart the service.
Sorry for the inconvenience.

**Reply by Striker62** [reply]
no problem if there is a solution . to edit a file is ok.
